### PR TITLE
Add `random` alias to `aesara.tensor.random.basic`

### LINKED ISSUE #1353: Add random `RandomVariable`
https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.random.html

### PR DESCRIPTION
**Pull request by redbopo**
Close #1353 

This PR will add random() function in RandomStream.
For the testsuite I just copy that as standard_normal() does, look for some feedback.

### CODE REVIEW

File: aesara/tensor/random/basic.py [hunk 1]
```diff
@@ -2086,6 +2086,20 @@ def __call__(self, x, **kwargs):
...
+    dtype = "floatX"
+    ndim_supp = 0
+    ndims_params = [0, 0]
+    _print_name = ("random", "\\operatorname{U}")
+
+    def __call__(self, size=None, **kwargs):
+        return super().__call__(size=size, **kwargs)
+
+
+random = RandomRV()
```
Comment (brandonwillard) [inline aesara/tensor/random/basic.py:2100]: In this case, we need to reuse the existing `UniformRV` `Op`.  Something like `random = uniform` might work.

Comment (redbopo) [reply aesara/tensor/random/basic.py]: Fixed, Thanks.

**Review by brandonwillard — APPROVED** [review]

**Comment by brandonwillard** [comment]
[The current error](https://github.com/aesara-devs/aesara/actions/runs/3699762886/jobs/6267501681#step:6:678) looks like it might be due to recent changes in JAX.

**Comment by rlouf** [comment]
> [The current error](https://github.com/aesara-devs/aesara/actions/runs/3699762886/jobs/6267501681#step:6:678) looks like it might be due to recent changes in JAX.

Yes. Pushed a fix in #1372

**Review by rlouf — APPROVED** [review]

**Pull request merged**